data/dataPaths.json

```diff
@@ -1010,14 +1010,21 @@
       "version": "bedrock/1.0"
     },
     "1.16.201": {
-      "protocol": "bedrock/1.16.201"
+      "protocol": "bedrock/1.16.201",
+      "steve": "bedrock/1.16.201",
+      "steveSkin": "bedrock/1.16.201",
+      "steveGeometry": "bedrock/1.16.201"
     },
     "1.16.210": {
-      "protocol": "bedrock/1.16.210"
+      "protocol": "bedrock/1.16.210",
+      "steve": "bedrock/1.16.201",
+      "steveSkin": "bedrock/1.16.201",
+      "steveGeometry": "bedrock/1.16.201"
     },
     "1.16.220": {
       "blockStates": "bedrock/1.16.220",
       "steve": "bedrock/1.16.201",
+      "steveSkin": "bedrock/1.16.201",
```

Comment:
should we really have this file in mcdata ? what is it used for ?

if we want to have it, then we need to change node-minecraft-data to support non json files

Reply:
So these are skin files used by the bedrock-protocol client when sending the login packet https://github.com/PrismarineJS/bedrock-protocol/blob/master/src/handshake/login.js#L10. So I think it would be possible to put it all into one big JSON with base64 if necessary

Reply:
yeah that would be more convenient

Reply:
Done, tests are failing now with the removal but should be fixed on node-minecraft-data release